lemon/api.py

```diff
@@ -42,21 +54,31 @@ def market_data(self) -> MarketDataAPI:
     def trading(self) -> TradingAPI:
         return self._trading
 
+    @property
+    def streaming(self) -> StreamingAPI:
+        return self._streaming
+
 
 def create(
     market_data_api_token: str,
     trading_api_token: str,
-    env: Literal["paper", "money"] = "paper",
+    env: Literal["paper", "money", "live"] = "paper",
     timeout: float = 5,
     retry_count: int = 3,
     retry_backoff_factor: float = 0.1,
 ) -> Api:
+    if env == "money":
+        warnings.warn(
+            "Usage of 'money' as env is deprecated, please use 'live' in the future",
+            category=FutureWarning,
```

Comment:
```suggestion
            category=DeprecationWarning,
```

Reply:
Thanks for pointing out that `DeprecationWarning` is not output on non __main__ execution.
Ignore my comment